src/styles/index.scss

```diff
@@ -2,2 +1,75 @@
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
+);
+$gap: 1px;
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $cell-size + 6 * $gap);
```

Comment:
The width calculation for `.calendar` should include the gap size for both sides of the cell. Right now, it only accounts for one side, which could lead to incorrect sizing. To fix this, you should use `7 * ($cell-size + $gap) - $gap`.